docs/root/configuration/operations/overload_manager/overload_manager.rst

```diff
@@ -353,3 +353,13 @@ with the following statistics:
 
   active, Gauge, "Active state of the action (0=scaling, 1=saturated)"
   scale_percent, Gauge, "Scaled value of the action as a percent (0-99=scaling, 100=saturated)"
+
+Each configured Load Shed Point has a statistics tree rooted at *overload.<name>.*
```

Comment:
```suggestion
Each configured Load Shed Point has a statistics tree rooted at ``overload.<name>.``
```

Reply:
Sorry, my bad. Should've left it to you @phlax

Reply:
:'( , I can open another PR to fix this if you want

Reply:
np - its up to you - i was checking the adjacent preambles and one did/not use literals - would be good to cleanup but not a huge priority - i would like to go through and just do a sweep of them to prevent copypasta etc